### DISCUSSION TITLE
Configurazione IKEA E1525/E1745 zigbee EZSP 6.7.6.0

### DISCUSSION THREAD

**Discussion by Virgilio123** [indomus/forum#2282] [ZigBee, Z-Wave, Thread, Matter]
Buon anno nuovo a tutti.

ho configurato un Sonoff zigBee Bridge utilizzando Tasmota versione 12.3.1 e come firmaware zigbee utilizzo EZSP 6.7.6.0

Riesco a collegare un Ikea Motion utilizzando la procedura prevista e nella console ricevo correttamente questo messaggio:
`23:36:55.267 MQT: tele/RouterZBBridge/SENSOR = {"ZbReceived":{"0xA723":{"Device":"0xA723","0006!42":"0008070000","Power":1,"PowerOnlyWhenOn":0,"PowerOnTime":180,"PowerOffWait":0,"Endpoint":1,"LinkQuality":97}}}`

Però non rilevo nessun movimento!!!

Ho visto in rete che anche altri utenti hanno lo stesso problema ma non sono riuscito a trovare nessun aiuto per risolvere.

Mi potete aiutare?

Grazie.

**Comment by snakuzzo** [comment]
Purtroppo no.
Se non viene pubblicato nulla sul broker, dovresti provare ad aprire una issue sul progetto Tasmota...inoltre non vedo alcuna issue sul progetto relativa al device in oggetto.
